src/i18n/hi/nav.ts

```diff
@@ -1,11 +1,74 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	// Start Here
 	startHere: 'यहाँ से शुरू करे',
 	'getting-started': 'पहले कदम',
+	install: 'स्थापित करें',
+	'editor-setup': 'एडिटर सेटअप',
+	'guides/upgrade-to/v3': 'v3 में अपग्रेड करें',
 
-	// Core Concepts
-	coreConcepts: 'मूल अवधारणाएं',
+	coreConcepts: 'मूल अवधारणाएँ',
 	'concepts/why-astro': 'क्यों Astro',
+	'concepts/islands': 'Astro द्वीप',
+
+	tutorials: 'शिक्षण सत्र',
+	'blog-tutorial': 'ब्लॉग बनाएं',
+	'add-collections-tutorial': 'सामग्री संग्रह बढ़ाएं',
+	'add-transitions-tutorial': 'दृश्य परिसर बढ़ाएं',
+
+	basics: 'मौलिक अवधारणाएँ',
+	'core-concepts/project-structure': 'परियोजना संरचना',
+	'core-concepts/astro-components': 'घटक',
+	'core-concepts/astro-pages': 'पृष्ठ',
+	'core-concepts/layouts': 'नक़्शे',
+	'core-concepts/astro-syntax': 'Astro की सिंटैक्स',
+
+	builtins: 'सामग्री स्थापित',
+	'guides/content-collections': 'सामग्री संग्रह',
+	'guides/view-transitions': 'view transitions',
```

Comment:
Another page with Astro Release 3.5 [Prefetch](https://astro.build/blog/astro-350/#prefetch)
```suggestion
	builtins: 'बिल्ट-इंस ',
	'guides/content-collections': 'सामग्री संग्रह',
	'guides/view-transitions': 'व्यू-ट्रांज़िशन्स ',
	'guides/prefetch': 'प्रीफ़ेच',
```